packages/empty-state/README.md

```diff
@@ -33,6 +33,7 @@ import { BasicEmptyState } from '@leafygreen-ui/empty-state';
   title="No Results Found"
   description="Try adjusting your filters or search terms"
   graphic={<SvgOrImgComponent />}
+  badgeProps={{ variant: 'blue', children: 'Optional' }}
```

Comment:
I'm curious—we allow components for the other props, like primaryButton, so why did we make this an object?